lms/static/sass/views/_login-register.scss

```diff
@@ -378,13 +394,21 @@ $sm-btn-linkedin: #0077b5;
             @include left(0);
 
             position: absolute;
-            top: -1px;
+            top: 0;
+            bottom: 0;
             width: 30px;
-            bottom: -1px;
             background: $m-blue-d3;
             line-height: $third-party-button-height;
             text-align: center;
             color: $white;
+
+            img {
+                width: auto;
+                height: auto;
+                max-height: 1.4em;
+                max-width: 1.4em;
+                margin-top: -4px;
+            }
```

Comment:
This selector duplicates the values set in _account.scss. If you use a class you could reuser the styles with the duplication.